app/resources/admin.html.template

```diff
@@ -68,38 +68,97 @@
     //find all languages that the user specified with "Use?" checkboxes and
     //their associated titles.  Only allows titles from used languages.
     var used_languages = [];
-    var languages_to_titles = {}
+    var language_code_to_index_set = {};
```

Comment:
Is this related to "index"? Otherwise how about renaming it to just "used_language_set" or "language_code_set"?